mesonbuild/interpreter/primitives/string.py

```diff
@@ -108,6 +108,7 @@ def join_method(self, args: T.Tuple[T.List[str]], kwargs: TYPE_kwargs) -> str:
         return self.held_object.join(args[0])
 
     @noKwargs
+    @FeatureNew('str.replace', '0.58.0')
     @typed_pos_args('str.replace', str, str)
```

Comment:
my bad :)

Reply:
No no no, see my review below. This was correct when you implemented it in commit c83106ee38048acbe737ef112e8d51c9b5bd42d7.